### ISSUE TITLE
[rivertile] The case of number of views being less than the number of masters is not handled

### ISSUE THREAD

**Issue by lazy-dolphin** [ifreund/river#58]
In mentioned case, `river` falls back to `full` layout.

**Comment by Leon-Plickat** [comment]
Could probably be solved by a single conditional.

```zig
if ( master_amount > view_amount ) {
    master_amount = view_amount;
}
````

**Comment by ifreund** [comment]
fixed in 6e23344e06e0318948e49213099be437a2039dfa
